crates/cli/src/git_utils.rs

```diff
@@ -103,6 +103,18 @@ fn get_changed_files(diff: &Diff) -> anyhow::Result<HashMap<PathBuf, Vec<u32>>>
     Ok(res)
 }
 
+pub fn get_repository_url(path: &str) -> anyhow::Result<String> {
+    let repository_opt = Repository::init(path);
+    match repository_opt {
+        Ok(repository) => Ok(repository
+            .find_remote("origin")?
```

Comment:
I'm a little wary of hardcoding origin here, it'd be nice if we could perform some logic to see if there's only one remote & use that, otherwise use origin, and *then* error out

Reply:
This is already how we do it for diff-aware, it's just taking the same approach.